portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translated")
+public class TranslationServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String originalText = request.getParameter("text");
+    String languageCode = request.getParameter("languageCode");
+
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+    Translation translation = translate.translate(
+        originalText, Translate.TranslateOption.targetLanguage(languageCode)
+        );
+    String translatedText = translation.getTranslatedText();
+
+    response.setContentType("text/html; charset=UTF-8");
+    response.setCharacterEncoding("UTF-8");
+    response.getWriter().println(translatedText);
```

Comment:
nit* response.getWriter().println(translation.getTranslatedText());. translatedText is not used.

Reply:
So the current function "response.getWriter().println(translatedText);" I should not define a separate variable for "translation.getTranslatedText();" and just call it directly in the println function since it's not used elsewhere?